deploy/examples/toolbox.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       serviceAccountName: rook-ceph-default
       containers:
         - name: rook-ceph-tools
-          image: quay.io/ceph/ceph:v18.2.2
+          image: quay.io/ceph/ceph:v19
```

Comment:
Should we wait to change the example manifests until 19.2.0 is released? Or are we planning to wait to merge this PR until squid is GA?

Reply:
Good catch, I didn't mean to commit this change for the toolbox yet, just the cluster-test.yaml.